api/metadata/server.go

```diff
@@ -76,6 +76,10 @@ func (s *Server) load() error {
 			}
 			fdps, e := allDependency(fdp)
 			if e != nil {
+				if e == protoregistry.NotFound {
```

Comment:
It would be better to use errors.Is here

Reply:
Thanks @tonybase you're right, done